services/checks/checks.go

```diff
@@ -578,23 +590,63 @@ func (s *Service) processResults(ctx context.Context, check check.Check, target
 		return nil, err
 	}
 
-	env, err := starlark.NewEnv(check.Name, check.Script, funcs)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to prepare starlark environment")
-	}
-
 	l := s.l.WithFields(logrus.Fields{
 		"name":       check.Name,
 		"id":         resID,
 		"service_id": target.serviceID,
 	})
+
+	scriptInput, err := agentpb.MarshalActionQueryDocsResult(r)
+	if err != nil {
+		return nil, err
+	}
+
+	input := &StarlarkScriptData{
+		CheckName:   check.Name,
+		Script:      check.Script,
+		Funcs:       funcs,
+		PrintFn:     l.Debugln,
+		ScriptInput: scriptInput,
+	}
+
+	process := exec.Command("./bin/pmm-managed-starlark")
```

Comment:
keep getting an error saying `fork/exec ./bin/pmm-managed-starlark: no such file or directory`

Reply:
Just "pmm-managed-starlark" – it is in the $PATH